handlers/people_test.go

```diff
@@ -20,33 +20,39 @@ import (
 )
 
 func TestGetPersonByPuKey(t *testing.T) {
-	mockDb := mocks.NewDatabase(t)
-	pHandler := NewPeopleHandler(mockDb)
+	teardownSuite := SetupSuite(t)
+	defer teardownSuite(t)
+
+	pHandler := NewPeopleHandler(db.TestDB)
 	t.Run("should return person if present in db", func(t *testing.T) {
 		rr := httptest.NewRecorder()
 		handler := http.HandlerFunc(pHandler.GetPersonByPubkey)
 		person := db.Person{
-			ID:          1,
-			Uuid:        uuid.New().String(),
-			OwnerPubKey: "person-pub-key",
-			OwnerAlias:  "owner",
-			UniqueName:  "test_user",
-			Description: "test user",
+			ID:           104,
+			Uuid:         "person_104_uuid",
+			OwnerPubKey:  "person_104_pubkey",
+			OwnerAlias:   "owner",
+			UniqueName:   "test_user",
+			Description:  "test user",
+			Tags:         pq.StringArray{},
+			Extras:       db.PropertyMap{},
+			GithubIssues: db.PropertyMap{},
 		}
+		db.TestDB.CreateOrEditPerson(person)
+
 		rctx := chi.NewRouteContext()
-		rctx.URLParams.Add("pubkey", "person-pub-key")
-		req, err := http.NewRequestWithContext(context.WithValue(context.Background(), chi.RouteCtxKey, rctx), http.MethodGet, "/person/person-pub-key", nil)
+		rctx.URLParams.Add("pubkey", "person_104_pubkey")
```

Comment:
@MuhammadUmer44 please use the variable `person.OwnerPubkey` here, this will help for an easy change in the future, and you don't need to reuse the same string when it is already on a variable.
```sh
		rctx.URLParams.Add("pubkey", person.OwnerPubkey)
```

Reply:
> @MuhammadUmer44 please use the variable `person.OwnerPubkey` here, this will help for an easy change in the future, and you don't need to reuse the same string when it is already on a variable.
> 
> ```shell
> 		rctx.URLParams.Add("pubkey", person.OwnerPubkey)
> ```

also use the `person.OwnerPubkey` on the route.

Reply:
@elraphty Fixed.